UKHomeOffice/sgmr-service
# SPCR-548/SPCR-549 Add max length to vessel registration input

### AC / Description of changes
To fix the issue with failing reports on live, implement a maximum allowed characters in the ‘Registration number' field on the second page of the ‘save-pleasure-craft’ journey to 15 characters.

Given user is on the second page of the save-pleasure-craft journey 
When entering registration number 
Then stop users being able to enter more than 15 characters

### To Test
- Go to /pleasure-crafts
- Create a new PC
- Go to the second page
- > Registration field should have a character limit of 15
- > The hint text should read 'Enter your registration number with special characters and spaces removed'
- Try to type special characters or spaces
- > These characters should not type in the input
- Try to type more than 15 characters
- Save the PC
- Click to edit the PC
- > The character count should be correct e.g. 10 characters remaining if registration was 5 characters

### Notes

---

**Remember to merge to feature branches not `master`.**
